lib/src/value/calculation.dart

```diff
@@ -120,6 +122,228 @@ class SassCalculation extends Value {
     return SassCalculation._("max", args);
   }
 
+  /// Creates a `hypot()` calculation with the given [arguments].
+  ///
+  /// Each argument must be either a [SassNumber], a [SassCalculation], an
+  /// unquoted [SassString], a [CalculationOperation], or a
+  /// [CalculationInterpolation]. It must be passed at least one argument.
+  ///
+  /// This automatically simplifies the calculation, so it may return a
+  /// [SassNumber] rather than a [SassCalculation]. It throws an exception if it
+  /// can determine that the calculation will definitely produce invalid CSS.
+  static Value hypot(Iterable<Object> arguments) {
+    var args = _simplifyArguments(arguments);
+    if (args.isEmpty) {
+      throw ArgumentError("hypot() must have at least one argument.");
+    }
+    _verifyCompatibleNumbers(args);
+
+    var subtotal = 0.0;
+    var iterator = 0;
+    var first = arguments.elementAt(0);
```

Comment:
Instead of `arguments.elementAt(0)`, prefer `arguments.first`.

Also, if you do something like:
```dart
if (first is! SassNumber) return SassCalculation._("hypot", args);
```
here, then Dart's static analysis should recognize that `first` is a `SassNumber` from any point after, so you won't need to repeated `if (first is SassNumber)` checks

Reply:
Oh right, I thought this would return a CSS function if the first argument contained any operation but I see it works. thanks!